src/utils/checkOrResizeImage.js

```diff
@@ -0,0 +1,33 @@
+export const checkOrResizeImage = (
+  base64Str,
+  originalImage,
+  setUserImageUrl,
+  setUpload,
+  maxWidth = 200,
+  maxHeight = 200
+) => {
+  const canvas = document.createElement('canvas');
+  const canvasCtx = canvas.getContext('2d');
+  let width = originalImage.naturalWidth;
+  let height = originalImage.naturalHeight;
+  if (width >= height) {
```

Comment:
Can you refactor this part? Thanks.

Reply:
changed to nested if statement to  && operator